### DISCUSSION TITLE
A broken link in a comment

### DISCUSSION THREAD

**Discussion by berk-karaal** [encode/django-rest-framework#8348] [Potential Issue] [answered]
https://github.com/encode/django-rest-framework/blob/f378f98a401f28df4ef9bdaf3ee56a1017c195ab/rest_framework/renderers.py#L108
That timelessrepo link is broken for more than a year. I think it should be replaced.

Check on Web Archive: https://web.archive.org/web/20201211162555/http://timelessrepo.com/json-isnt-a-javascript-subset

**Comment by lovelydinosaur** [comment, marked as answer]
Thanks! Now resolved https://github.com/encode/django-rest-framework/commit/a5d741aba473d5099d59ff622dbf9702df842441
